apps/user/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.contrib.auth.models import AbstractUser
+from django_enumfield import enum
+
+
+class User(AbstractUser):
+    email = models.EmailField(unique=True)
+    username = models.CharField(
+        max_length=20,
+        help_text="Required 20 characters or fewer",
+    )
+
+    USERNAME_FIELD = 'email'
+    REQUIRED_FIELDS = ['username']
+
+    def get_full_name(self):
+        return f'{self.first_name} {self.last_name}'
+
+class Profile(models.Model):
+    """
+    User profile model
+    """
+
+    class GENDER(enum.Enum):
```

Comment:
We should move all this field to `User` if we are using a custom User model.
cc: @annshress

Reply:
@thenav56 has a point. If we are already writing our own User model. Maybe we can merge `Profile` model into `User` rather than a `one to one` relation.

@k9845

Reply:
@thenav56 has a point. If we are writing our own `User` model, merge `Profile` into the `User` to remove the redundant one-to-one relation.

@k9845